### PR TITLE
2021 annual report (QA through map section)

### PR DESCRIPTION
**Pull request by caseymm**
No big changes, just some edits to clean up the alignment of the sections from the top through the map section. These sections (only) are ready to go through QA.

### CODE REVIEW

File: src/views/annual-report/2021/l10n.json
Comment (rschamp) [inline src/views/annual-report/2021/l10n.json:55]: I am curious if this is the best way to do this, or if it will be problematic for the string to be split across two different messages.

**Review by rschamp — APPROVED** [review]
I am going to leave this to @seotts to merge, but it looks good to me pending one question about splitting up that l10n string.

**Comment by caseymm** [comment]
Added a few more things to be QAd. Feel free to take a look down through the "Volunteer Translators" image.

File: src/views/annual-report/2021/l10n.json
Comment (seotts) [reply src/views/annual-report/2021/l10n.json:55]: Actually, we probably want the placeholder name to reflect the content more than the styling (something like `numberOfCountries`.) I'll make that change after I merge this PR.

**Comment by seotts** [comment]
This looks good to merge, thank you!

**Pull request merged**
